pkg/controller/controller.go

```diff
@@ -97,16 +97,25 @@ func (c *Controller) initOperatorConfig() {
 		c.logger.Infoln("no ConfigMap specified. Loading default values")
 	}
 
-	// env var takes priority over the same param from the operator ConfigMap
-	watchedNamespace := os.Getenv("WATCHED_NAMESPACE")
-	if watchedNamespace != "" {
-		c.logger.Infof("Watch the %q namespace specified in the env variable WATCHED_NAMESPACE\n", watchedNamespace)
-		configMapData["watched_namespace"] = watchedNamespace
+	// by default, the operator listens to all namespaces
+	if configMapData["watched_namespace"] == "" {
```

Comment:
Just to confirm: does this behave as expected, given that it will behave the same way when the key is missing and when it is explicitly set to the empty string?